src/components/molecules/main-nav/main-nav.tsx

```diff
@@ -60,18 +70,18 @@ const MainNav = ({ navList }: MainNavProps) => {
                 {navList.map((navItem, index) => (
                   <li
                     key={index}
-                    className={`flex items-center justify-end gap-2 py-2 ${
+                    className={`flex items-center justify-end space-x-2 py-2 text-right ${
                       index !== 0 && "border-t"
                     }`}
                   >
-                    <NavLink {...navItem} />
                     {navItem.externalLink && (
                       <Icon
                         Icon={FiExternalLink}
                         size={22}
                         iconAlt="external link for TechIsHiring newsletter"
                       />
                     )}
```

Comment:
@chadstewart This is the code block you are asking for. I never deleted it, it is only visible in the drawer component

Reply:
This is not the code block you are talking about so I will close this conversation